File: Web/KombiCim.Data/Models/Arduino/Dtos/ProfileDto.cs
using Newtonsoft.Json;

namespace Kombicim.Data.Models.Arduino.Dtos
{
    public class ProfileDto
    {
        public int Id { get; set; }
        public string ProfileName { get; set; }
        public string TypeName { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public bool? State { get; set; }
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public double? MinTempValue { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public LocationDto SelectedThermometer { get; set; }

        public bool Active { get; set; }
    }
}

File: Android/app/src/main/java/com/kc/kombicim/Model/Dto/ProfileDto.kt
package com.kc.kombicim.Model.Dto

import com.google.gson.annotations.SerializedName

data class ProfileDto (

    @SerializedName("Id") val id : Int,
    @SerializedName("ProfileName") val profileName : String,
    @SerializedName("TypeName") val typeName : String,
    @SerializedName("MinTempValue") val minTempValue : Double?,
    @SerializedName("State") val state : Boolean?,
    @SerializedName("Active") val active : Boolean,
    @SerializedName("SelectedThermometer") val selectedThermometer: LocationDto?
)